modules/core/04-channel/v2/keeper/msg_server.go

```diff
@@ -47,8 +48,34 @@ func (*Keeper) Acknowledgement(ctx context.Context, acknowledgement *channeltype
 }
 
 // RecvPacket implements the PacketMsgServer RecvPacket method.
-func (*Keeper) RecvPacket(ctx context.Context, packet *channeltypesv2.MsgRecvPacket) (*channeltypesv2.MsgRecvPacketResponse, error) {
-	panic("implement me")
+func (k *Keeper) RecvPacket(ctx context.Context, msg *channeltypesv2.MsgRecvPacket) (*channeltypesv2.MsgRecvPacketResponse, error) {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	packet := msg.Packet
```

Comment:
[nit] I think we typically just access `msg.Packet` when needed rather than storing a separate var here, don't think this is required.

Reply:
Yeah this was supposed to be "temporary" and I forgot, will remove!